crypter.go

```diff
@@ -108,6 +108,8 @@ type Recipient struct {
 	Algorithm KeyAlgorithm
 	Key       interface{}
 	KeyID     string
+	P2C       int
+	P2S       []byte
```

Comment:
Can we change the names of these fields to make it clear what they do? Or, alternatively, add doc comments.

Reply:
Alternative names could be PBES2Count and PBES2Salt, or just Count and Salt. But I rather add docs and have the field names match better the data written in the header.

Reply:
I like `PBES2Count` and `PBES2Salt`. Since this struct doesn't directly represent a header I think it's ok to to have names not match the header fields.

Reply:
Ok, I'll change it